org.opent2t.sample.hub.superpopular/com.insteon.hub/js/thingTranslator.js

```diff
@@ -0,0 +1,300 @@
+﻿/* jshint esversion: 6 */
+/* jshint node: true */
+/* jshint sub:true */
```

Comment:
Included to be consistent other hub translators. Arjun commented that we should keep them consistent and remove the comments all at once when we decide they are not needed.